official/mnist/mnist_tpu.py

```diff
@@ -82,13 +83,21 @@ def model_fn(features, labels, mode, params):
   """model_fn constructs the ML model used to predict handwritten digits."""
 
   del params
-  if mode == tf.estimator.ModeKeys.PREDICT:
-    raise RuntimeError("mode {} is not supported yet".format(mode))
   image = features
   if isinstance(image, dict):
     image = features["image"]
 
   model = mnist.create_model("channels_last")
+
+  if mode == tf.estimator.ModeKeys.PREDICT:
+    logits = model(image, training=False)
+    predictions = {
+        'class_ids': tf.argmax(logits, axis=1),
+        'probabilities': tf.nn.softmax(logits),
+        'label': labels,
```

Comment:
The `PREDICT` model function shouldn't produce the training label, right? (How would one know that?)

Reply:
Done.